src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/SetAzureWebApp.cs

```diff
@@ -83,6 +83,9 @@ public class SetAzureWebAppCmdlet : WebAppBaseCmdlet
         [ValidateNotNullOrEmpty]
         public bool Use32BitWorkerProcess { get; set; }
 
+        [Parameter(Position = 15, Mandatory = false, HelpMessage = "Destination slot name for auto swap")]
```

Comment:
15?  15 is way too many positional parameters.  Ideally you would remove all but 3 or 4, but at least don't add a 15th :-)